config.js

```diff
@@ -240,10 +216,27 @@ exports.updateFromArgv = function() {
                 + `\nDebug status: ${debugStatus}`);
     // B - check unsupported argv //
     testDroppedArgv(argv);
+    checkUnsupportedOgspvAI(argv.ogspv, ogsPvAIs);
```

Comment:
You are no longer calling `toUpperCase` on `argv.ogspv` like you did before. This is okay, as it's a minor detail not needed here, but does mean `checkUnsupportedOgspvAI` has to be modified to include this.

Reply:
yes :+1:  fixed